packages/react-devtools-shared/src/devtools/views/Profiler/CommitRankedListItem.js

```diff
@@ -52,6 +77,8 @@ function CommitRankedListItem({data, index, style}: Props) {
       key={node.id}
       label={node.label}
       onClick={handleClick}
+      onMouseOver={event => handleMouseOver(event, node)}
```

Comment:
As `node` can be accessed from closure, maybe 
```suggestion
      onMouseOver={handleMouseOver}
```